File: app/src/main/java/com/example/myapplication/fragments/anime_fragments/anime_new_releases_mvp/AnimeNewReleasesInterface.java
package com.example.myapplication.fragments.anime_fragments.anime_new_releases_mvp;

import com.example.myapplication.models.animemodels.AnimeNewReleaseResultModel;

import java.util.List;

public interface AnimeNewReleasesInterface {
    void onGetNewReleasesDataSuccess(List<AnimeNewReleaseResultModel> animeNewReleases, String hitStatus);

    void onGetNewReleasesDataFailed();
}

File: app/src/main/java/com/example/myapplication/fragments/anime_fragments/anime_new_releases_mvp/AnimeNewReleasesInterface.kt
package com.example.myapplication.fragments.anime_fragments.anime_new_releases_mvp

import com.example.myapplication.models.animemodels.AnimeNewReleaseResultModel

interface AnimeNewReleasesInterface {
    fun onGetNewReleasesDataSuccess(animeNewReleases: List<AnimeNewReleaseResultModel>, hitStatus: String)

    fun onGetNewReleasesDataFailed()
}
